resources/api/openapi.yml

```
@@ -389,6 +406,36 @@ components:
         - dates
         - language
         - arrived
+    WorkLog:
```

```suggestion
    Worklog:
```
Everywhere else it's "Worklog"